src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/DependencyAnalysis/Target_ARM/ARMReadyToRunHelperNode.cs

```diff
@@ -128,7 +131,7 @@ protected override void EmitCode(NodeFactory factory, ref ARMEmitter encoder, bo
                         if (targetMethod.OwningType.IsInterface)
                         {
                             encoder.EmitMOV(encoder.TargetRegister.Arg1, factory.InterfaceDispatchCell(targetMethod));
-                            encoder.EmitJMP(factory.ExternFunctionSymbol("RhpResolveInterfaceMethod"));
+                            encoder.EmitJMP(factory.ExternFunctionSymbol(new Utf8String("RhpResolveInterfaceMethod"u8)));
```

Comment:
These (`RhpResolveInterfaceMethod` in XXXReadyToRunHelperNode.cs) might be worth extracting into a `static readonly Utf8String`. 

The `new Utf8String("SomeLiteral"u8)` pattern is repeated elsewhere (each of them is an array allocation). I don't have a good sense for which ones should be extracted to readonly statics, but this one might be called often enough to be worth it.